docs/src/main/asciidoc/mp/server.adoc

```diff
@@ -312,6 +312,20 @@ io.helidon.examples.AdminService:
     path: "/management"
 ----
 
+The same example configuration for `.properties` file:
```

Comment:
I think information about setting the JAX-RS application's context path should go in `mp/jaxrs/jaxrs-applications.adoc`.  Maybe add a new section after "Discovery of REST Beans" since it mentions `@ApplicationPath`.  Maybe:

1. Add a new section "Setting Application Path" (mention in description this is also know as context path)
2. Show an example of how to do it using `@ApplicationPath` (don't use AdminService, use something like MyApplication).
3. Show how to override that using configuration (when you have an Application class)
4. Show how to configure it if there is no Application class

Keep in mind the question we are trying to answer is "How do I set the context path of my JAX-RS application".  So it shouldn't be buried in the Helidon server routing feature.